### PR TITLE
Added RT conflict

### PR DESCRIPTION
**Pull request by linuxgurugamer**
Per request

### CODE REVIEW

**Comment by linuxgurugamer** [comment]
The error shown makes no sense to me.  Theonly change was adding the conflict to RT.
If I made a mistake, please explain it to me.
Thanks

File: NetKAN/AntennaRangePatches.netkan [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		{ "name": "RemoteTech"}
+	],
+	"install": [
+		{
+			"find"		: "AntennaRangePatches",
+			"install_to"	: "GameData"
+		},
+		{
+			"find"		: "AutoPruner",
+			"install_to"	: "GameRoot"
```
Comment (plague006) [inline NetKAN/AntennaRangePatches.netkan]: As I told you in #2427, CKAN will not install external tools.

**Comment by plague006** [comment]
Closing as duplicate of #2460.

File: NetKAN/AntennaRangePatches.netkan [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		}
+	],
+	"recommends": [
+		{ "name": "RemoteTech" },
```
Comment (plague006) [inline NetKAN/AntennaRangePatches.netkan]: You cannot recommend a mod you conflict with.

Comment (linuxgurugamer) [inline NetKAN/AntennaRangePatches.netkan]: ok.

I goofed, sorry

now I'm confused.

is this already fixed and committed?

Before I do another pull request?  i don't see it there.

I've removed the AutoPruner and removed the recommends RT line, so that 
should be ok.

Thanks

JBB
